Sources/Tentacle/Homepage.swift

```diff
@@ -0,0 +1,28 @@
+//
+//  Homepage.swift
+//  Tentacle
+//
+//  Created by Romain Pouclet on 2017-06-20.
+//  Copyright © 2017 Matt Diephouse. All rights reserved.
+//
+
+import Foundation
+
+public enum Homepage: Decodable {
```

Comment:
What is the purpose of this type? Keep using `URL` seems better to me.

Reply:
Unless the API is expected to respond with an arbitrary String and not a valid URL, I agree.

Reply:
I understand now that the problem here is empty string handling (`""`). Empty string is not valid input even if a property is `URL?`.

Reply:
In that case it seems like we should revert to `URL?` and change the decoding?

Reply:
GitHub will let you set this field to any string.

BUT your github.com profile will only show the string if it's actually a URL. So I think we could revert to a `URL` and just set it to `nil` if it's not a valid URL.

Reply:
Pushed aa43141 to address this.

Reply:
Oops, missed this, removed my comment 🙊
Are we sure it’s the same with a repository? I remember having this conversation a while ago 🤔